server/app/services/applicant/ApplicantService.java

```diff
@@ -536,20 +536,20 @@ CompletionStage<Optional<ApplicationModel>> savePrimaryApplicantInfoAnswers(
                                       applicant.setEmailAddress(
                                           applicantData
                                               .readString(path.join(Scalar.EMAIL))
-                                              .orElseThrow());
+                                              .orElse(""));
```

Comment:
Values that are empty or blank are saved as `null` to the db (Eg [for email](https://github.com/civiform/civiform/blob/cb4289acf3e8b1c90d7bfe23d299a9ac356f936c/server/app/models/ApplicantModel.java#L170)) so this does not cause a issue with actions that rely on PAI info (eg emailing status updates).